static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -313,6 +319,8 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
       el.classList.remove('yxt-Card--pinFocused');
     });
 
+    this._detailCard && this._detailCard.remove();
```

Comment:
Should we update the name of this method? Now, the card is actually getting removed, correct? The method name suggests we're just toggling CSS classes/focus states.

Reply:
updated!